File: tutorials/csharp-tutorial/src/CSharpTutorialDemo/CSharpTutorialDemo/ClassesAndObjects.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace CSharpTutorialDemo
{
    public class ClassesAndObjects
    {
        public class CustomerInfo
        {
            public int Id { get; set; }
            public string Name { get; set; }
            public string Address { get; set; }

            public CustomerInfo()
            {
            }

            public CustomerInfo(int id, string name, string address)
            {
                this.Id = id;
                this.Name = name;
                this.Address = address;
            }

            public void Print()
            {
                Console.WriteLine("Id: {0}, Name: {1}, Address: {2}", this.Id, this.Name, this.Address);
            }
        }

        public static void Example1()
        {
            CustomerInfo customer = new CustomerInfo();

            customer.Id = 1;
            customer.Name = "Andy";
            customer.Address = "22 wall street";

            customer.Print();
        }

        public static void Example2()
        {
            CustomerInfo customer = new CustomerInfo(2, "John", "11 wall street");

            customer.Print();
        }
    }
}

File: tutorials/vb-net/src/VBNetTutorialDemo/VBNetTutorialDemo/ClassesAndObjects.vb
Public Module ClassesAndObjects
    Public Class Customer
        Public Property Id As Integer
        Public Property Name As String
        Public Property Address As String

        Public Sub New()
        End Sub

        Public Sub New(ByVal id As Integer, ByVal name As String, ByVal address As String)
            Me.Id = id
            Me.Name = name
            Me.Address = address
        End Sub

        Public Sub Print()
            Console.WriteLine("Id: {0}, Name: {1}, Address: {2}", Me.Id, Me.Name, Me.Address)
        End Sub
    End Class

    Sub Example1()
        Dim customer As Customer = New Customer()
        customer.Id = 1
        customer.Name = "Andy"
        customer.Address = "22 wall street"

        customer.Print()
    End Sub

    Sub Example2()
        Dim customer As Customer = New Customer(2, "John", "11 wall street")
        customer.Print()
    End Sub

End Module
